node/src/components/consensus/protocols/highway/synchronizer/tests.rs

```diff
@@ -67,7 +67,10 @@ fn purge_vertices() {
     assert!(maybe_pv.is_none());
     assert!(matches!(
         *outcomes,
-        [ProtocolOutcome::CreatedTargetedMessage(_, NodeId(0))]
+        [
+            ProtocolOutcome::CreatedTargetedMessage(_, NodeId(0)),
+            ProtocolOutcome::CreatedTargetedMessage(_, NodeId(0))
```

Comment:
It would be good to check what we're requesting here. Maybe re-create the message and serialize it, so we can make the full comparison?